observability/src/test/java/io/grpc/observability/LoggingChannelProviderTest.java

```diff
@@ -53,9 +53,10 @@
 
   @Test
   public void initTwiceCausesException() {
+    ManagedChannelProvider prevProvider = ManagedChannelProvider.provider();
```

Comment:
Here we need assert `preProvider` not `instanceof LoggingChannelProvider`, to be as strong as to previous version.

Reply:
done